### ISSUE TITLE
Doesn't build against uri 2.0.0

### ISSUE THREAD

**Issue by gtrak** [brendanlong/sentry-ocaml#13]
```
#=== ERROR while compiling sentry.~dev ========================================#
# context     2.0.1 | linux/x86_64 | ocaml-base-compiler.4.06.1 | pinned(git+https://github.com/brendanlong/sentry-ocaml.git#v0.4.1#d866de1b)
# path        ~/.opam/4.06.1/.opam-switch/build/sentry.~dev
# command     ~/.opam/opam-init/hooks/sandbox.sh build jbuilder build -p sentry -j 3
# exit-code   1
# env-file    ~/.opam/log/sentry-19599-d2c37b.env
# output-file ~/.opam/log/sentry-19599-d2c37b.out
### output ###
#     ocamlopt src/.sentry.objs/sentry__Dsn.{cmx,o} (exit 2)
# (cd _build/default && /home/gary/.opam/4.06.1/bin/ocamlopt.opt -w -40 -g -I src/.sentry.objs -I src/.sentry.objs/.private -I /home/gary/.opam/4.06.1/lib/astring -I /home/gary/.opam/4.06.1/lib/async -I /home/gary/.opam/4.06.1/lib/async_extra -I /home/gary/.opam/4.06.1/lib/async_kernel -I /home/gary/.opam/4.06.1/lib/async_rpc_kernel -I /home/gary/.opam/4.06.1/lib/async_ssl -I /home/gary/.opam/4[...]
# File "src/dsn.ml", line 4, characters 10-15:
# Error: Unbound value Uri.sexp_of_t
#       ocamlc src/.sentry.objs/sentry__Dsn.{cmo,cmt} (exit 2)
# (cd _build/default && /home/gary/.opam/4.06.1/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.sentry.objs -I src/.sentry.objs/.private -I /home/gary/.opam/4.06.1/lib/astring -I /home/gary/.opam/4.06.1/lib/async -I /home/gary/.opam/4.06.1/lib/async_extra -I /home/gary/.opam/4.06.1/lib/async_kernel -I /home/gary/.opam/4.06.1/lib/async_rpc_kernel -I /home/gary/.opam/4.06.1/lib/async_ssl -I /home/gar[...]
# File "src/dsn.ml", line 4, characters 10-15:
# Error: Unbound value Uri.sexp_of_t
```

**Comment by gtrak** [comment]
Due to https://github.com/ocaml/opam-repository/pull/12808

**Comment by brendanlong** [comment]
This should be fixed in 4d485edaabdb132ed2b9d12165bc8782c75ad7c9
